FlowStackExample/FlowStackExample.xcodeproj/project.pbxproj

```diff
@@ -309,15 +316,15 @@
 				CODE_SIGN_STYLE = Automatic;
 				CURRENT_PROJECT_VERSION = 1;
 				DEVELOPMENT_ASSET_PATHS = "\"FlowStackExample/Preview Content\"";
-				DEVELOPMENT_TEAM = CBXAQN69JX;
+				DEVELOPMENT_TEAM = L979M8V7RW;
```

Comment:
☝️ Which team did we have previously and which team is this? I think we probably want to revert this as well.